analyzers/src/SonarAnalyzer.CSharp/Rules/DoNotDecreaseMemberVisibility.cs

```diff
@@ -135,28 +127,30 @@ private Diagnostic FindMethodIssue(MemberDeclarationSyntax memberDeclaration, IM
 
             private Diagnostic FindPropertyIssue(MemberDeclarationSyntax memberDeclaration, IPropertySymbol propertySymbol)
```

Comment:
FindMethodIssue and FindPopertyIssue methods are very similar, maybe they could be unified?

Reply:
Indeed they look very similar but I would let this exercise for the next time we will touch the class.